### PR TITLE
openvpn-easy-rsa: bump to 3.0.7

### PR DESCRIPTION
**Pull request by luizluca**
New easyrsa will look for missing vars and x509-types where easyrsa
is located (following symlink). /usr/bin/easyrsa is now a link
to /usr/lib/easyrsa/easyrsa and /usr/lib/easyrsa/{vars,x509-types} a
link to /etc/easyrsa/{vars,x509-types}. This keeps the same previous
OpenWrt easyrsa behavior which tries to use $PWD/pki and
/etc/easyrsa/{vars,x509-types}, but without patching it.

Easyrsa can also use env vars to set pki root path (instead of
/usr/lib/easyrsa), pki path (instead of $PWD/pki) and vars path.
Those variables are commented in /etc/profile.d/50-openvpn-easy-rsa.sh
as an example of how to make easyrsa run independent of $PWD. That
scriptlet also sets $EASYRSA_TEMP_DIR from $EASYRSA_PKI/tmp to /tmp
in order to avoid writing to persistent media (normally flash). However,
as a profile scriptlet, it will only be used after session is restarted.

The "build" tgz was replaced by the "source" tar. "build" version has a
different file structure, making any patch backports too complex.

Signed-off-by: Luiz Angelo Daros de Luca <luizluca@gmail.com>

### CODE REVIEW

File: package/network/services/openvpn-easy-rsa/Makefile [hunk 1]
```diff
@@ -9,14 +9,22 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openvpn-easy-rsa
 
-PKG_VERSION:=3.0.4
-PKG_RELEASE:=1
-PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa/releases/download/v$(PKG_VERSION)/
-PKG_SOURCE:=EasyRSA-$(PKG_VERSION).tgz
-PKG_HASH:=472167f976c6cb7c860cec6150a5616e163ae20365c81f179811d6ee0779ec5a
+PKG_VERSION:=3.0.7
+KG_RELEASE:=1
```
Comment (hauke) [inline package/network/services/openvpn-easy-rsa/Makefile:13]: This misses a P

**Review by hauke — APPROVED** [review]

File: package/network/services/openvpn-easy-rsa/Makefile (hunk 1 above)
Comment (luizluca) [reply package/network/services/openvpn-easy-rsa/Makefile]: Yes, wrong vi command. Fixed. Thanks.

File: package/network/services/openvpn-easy-rsa/Makefile [hunk 2]
```diff
@@ -9,14 +9,22 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openvpn-easy-rsa
 
-PKG_VERSION:=3.0.4
+PKG_VERSION:=3.0.7
 PKG_RELEASE:=1
-PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa/releases/download/v$(PKG_VERSION)/
-PKG_SOURCE:=EasyRSA-$(PKG_VERSION).tgz
-PKG_HASH:=472167f976c6cb7c860cec6150a5616e163ae20365c81f179811d6ee0779ec5a
+PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa/archive/
```
Comment (neheb) [inline package/network/services/openvpn-easy-rsa/Makefile:14]: No. This filename is completely useless and potentially conflicts. Use codeload.

See the packages feed as an example.

**Comment by adrianschmutzler** [comment]
@luizluca Ping.

**Comment by neheb** [comment]
If I had to guess, @ynezz wants it out of here.

File: package/network/services/openvpn-easy-rsa/Makefile (hunk 2 above)
Comment (luizluca) [reply package/network/services/openvpn-easy-rsa/Makefile]: Done!

**Comment by luizluca** [comment]
@neheb , now I'm using codeload. Thanks for the review.

**Pull request closed without merging**